README.md

```diff
@@ -1,2 +1,5 @@
-# Project-Trial-Review
-This  project is to review my previous project I had completed during the trials.
```

Comment:
Please make your readme file more descriptive. You should try adding more sections to the readme like what your skills are , your contacts and social media .